tuf_conformance/client_runner.py

```diff
@@ -120,3 +120,23 @@ def trusted_roles(self) -> list[tuple[str, int]]:
             roles.append((rolename, md.signed.version))
 
         return roles
+
+    def assert_metadata(self, role: str, expected_bytes: bytes | None) -> None:
```

Comment:
this is a little different from trusted_roles() and version() -- as those do not assert, they only return the current value. This felt easier to use correctly in tests but I could change it.

Reply:
I think it is okay for now. I am fine with changing later if we see an issue here.